lib/lint.js

```diff
@@ -77,6 +77,7 @@ module.exports = ({ args }) => {
       parser: 'babel-eslint',
     },
   })
+  console.log('running eslint cli')
```

Comment:
Do we want this in?

Reply:
the log statement is necessary b/c otherwise console output may be misleading. for example, eslint prints "rerun the command with --fix" and users may erroneously think that they should run ace lint --fix which does not work.

Reply:
Can we suppress the eslint output?

Reply:
we print the output here: https://github.com/connexta/ace/pull/11/files#diff-71176734a4eda13ab0ff480990387537R84
we could not print it or filter out the --fix message, but I recommend leaving the output as is